tests/integration/ceph_base_file_test.go

```diff
@@ -265,6 +266,132 @@ func runFileE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.S
 	assert.NoError(s.T(), err)
 	downscaleMetadataServers(helper, k8sh, s, settings.Namespace, filesystemName)
 	cleanupFilesystem(helper, k8sh, s, settings.Namespace, filesystemName)
+=======
+	t := s.T()
```

Comment:
Looks like I should finish the backport #10471 to allow this to merge more cleanly.